openedx/features/ucsd_features/message_types.py

```diff
@@ -0,0 +1,8 @@
+from openedx.core.djangoapps.ace_common.message import BaseMessageType
+
+
+class SupportNotification(BaseMessageType):
```

Comment:
can we use `BaseMessageType` directly?

Reply:
No, as templates will be selected on the basis of these message types.